src/test/test_interval_bitwise.cpp

```
@@ -23,6 +23,13 @@ TEST_CASE("bitwise_and with non-singleton containing all-ones includes zero", "[
     REQUIRE(left.bitwise_and(anomalous_right) == Interval{0, 100});
 }
 
+TEST_CASE("signed division by a negative singleton preserves interval order", "[interval][arithmetic]") {
+    REQUIRE(Interval{-8, -4}.sdiv(Interval{-1}) == Interval{4, 8});
+    REQUIRE(Interval{4, 8}.sdiv(Interval{-2}) == Interval{-4, -2});
```

Cppcheck (2.20.0)

[style] The function 'make_jump' is never used.

(unusedFunction)